src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1053,7 +1056,7 @@ export default class DefaultDeviceController
       this.deviceInfoCache = [];
       return;
     }
-
+    this.logger.debug('Update device info cache');
```

Comment:
Should we move this below after getting the devices and also print the `devices`, because we will not know updated devices until builders implement `audioInputsChanged` observer then and hence might miss in the logs.

Reply:
Updated